src/main/java/com/cinema/Main.java

```diff
@@ -80,38 +82,38 @@ public static void main(String[] args) throws AuthenticationException {
         userAlice.setPassword("alice's_pass");
         userService.add(userAlice);
 
-        System.out.println("User by email: " + userBob.getEmail());
-        System.out.println(userService.findByEmail("bob@gmail.com"));
+        logger.info("User by email: " + userBob.getEmail());
+        logger.info(userService.findByEmail("bob@gmail.com"));
 
         AuthenticationService authenticationService =
                 (AuthenticationService) injector.getInstance(AuthenticationService.class);
 
         User registeredUser = authenticationService
                 .register("newUserEmail@mail.com", "!pass_*_pass!");
 
-        System.out.println("Trying to login user:");
-        System.out.println(authenticationService
+        logger.info("Trying to login user:");
+        logger.info(authenticationService
```

Comment:
use try-catch

Reply:
It is valid password, so here won't be any exceptions. But I changed message for clearer understanding, ty